app/plans/get.controller.js

```diff
@@ -1,27 +1,27 @@
 const { getSentencePlanSummary } = require('../../common/data/sentencePlanningApi')
 const { isEmptyObject } = require('../../common/utils/util')
 
-const hasActivePlan = plans => {
-  if (isEmptyObject(plans)) return false
-  let activePlanPresent = false
-  plans.forEach(plan => {
-    if (!plan.completedDate) {
-      activePlanPresent = true
-    }
-  })
-  return activePlanPresent
+const getPlan = plans => {
+  if (isEmptyObject(plans)) return {}
+  const [thePlan] = plans.filter(plan => plan.completedDate === '' || plan.completedDate === undefined)
```

Comment:
Why the square brackets?

Reply:
Also deconstruct rather than referencing twice.
```
const [thePlan] = plans.filter(( { completedDate } )=> completedDate === '' || completedDate === undefined)
```

Reply:
Following discussions with @Coupey, rather than de-structure the array from the filter, if you know you only want one item then use a find.